libs/coin-evm/src/api/node/ledger.ts

```diff
@@ -272,17 +272,36 @@ export const getBlockByHeight: NodeApi["getBlockByHeight"] = async (
     throw new LedgerNodeUsedIncorrectly();
   }
 
-  const { hash, height, time } = await fetchWithRetries<{
-    hash: string;
-    height: number;
-    time: string;
-    txs: string[];
-  }>({
+  if (blockHeight === "latest") {
```

Comment:
@chabroA Has this been discussed with Backend Blockchain team? Unless there's a reason (which I doubt) it seems like it would better be fixed on backend side with a unique format.

Reply:
After discussion, let's merge this then plan for later a proper backend fix and a rework of bitcoin & evm.